install/generator/04-syndesis-server.yml.mustache

```diff
@@ -126,6 +126,13 @@
               path: "/health"
               port: 8181
             initialDelaySeconds: 10
+          resources:
```

Comment:
why is this section added here as it is already included some lines before ? If its about adding cpu limit / resources please adapt the existing 'resources' configuration.

Reply:
@zregvart 
No problem. They are part of the commit.